src/pint/pintk/plk.py

```diff
@@ -914,6 +920,32 @@ def plotResiduals(self, keepAxes=False):
             for i in range(len(rs)):
                 self.plkAxes.plot(f_toas_plot, rs[i], "-k", alpha=0.3)
 
+    def determine_yaxis_units(self, maxy, miny):
```

Comment:
I was thinking since most of the other pintk-graph related code is in here, it made sense to put it here, but if you think pulsar.py or somewhere else would be better I'd be happy to relocate it. I wrote a test for it - see below - it does not need the actual GUI to test.